Source/JavaScriptCore/jsc.cpp

```diff
@@ -3013,10 +3019,10 @@ void CommandLine::parseArguments(int argc, char** argv)
 template<typename Func>
 int runJSC(const CommandLine& options, bool isWorker, const Func& func)
 {
-    printf("CHERI-jsc " CHERI_VERSION_FLAVOR " " CHERI_VERSION_BACKEND " " CHERI_VERSION_HEAP_OFFSET " \n");
+    printf("CHERI-jsc " CHERI_VERSION_FLAVOR " " CHERI_VERSION_BACKEND " " CHERI_VERSION_MASM_PROBE " " CHERI_VERSION_HEAP_OFFSET " \n");
```

Comment:
This is a very petty comment but on-non CHERI this will print `CHERI_VERSION_BACKEND " " " " CHERI_VERSION_HEAP_OFFSET` i.e. two spaces between backend/heap_offset.

Reply:
It also ends with a trailing space :-)

Tidying this up is perhaps a separate concern so I might make another PR, if that's Ok.

Reply:
Agreed!